rllab/misc/logger.py

```diff
@@ -3,6 +3,7 @@
 from rllab.misc.tabulate import tabulate
 from rllab.misc.console import mkdir_p, colorize
 from rllab.misc.autoargs import get_all_parameters
+from rllab.misc.tensor_summary import Summary
```

Comment:
please reorder imports to conform to PEP8

https://www.python.org/dev/peps/pep-0008/#imports